compliance/cis_k8s/rules/cis_4_2_3/rule.rego

```diff
@@ -4,19 +4,41 @@ import data.compliance.cis_k8s
 import data.compliance.lib.common
 import data.compliance.lib.data_adapter
 
+default rule_evaluation = false
+
+process_args := data_adapter.process_args
+
+contains_ca_file {
+	common.contains_key(process_args, "--client-ca-file")
+}
+
+rule_evaluation {
+	contains_ca_file
+}
+
+# In case both flags and configuration file are specified, the executable argument takes precedence.
+# Checks that the entry for authentication:x509:clientCAFile: set to a valid path.
+rule_evaluation {
+	not contains_ca_file
+	data_adapter.process_config.config.authentication.x509.clientCAFile
+	not data_adapter.process_config.config.authentication.x509.clientCAFile == ""
```

Comment:
what is this check based on? can the path be an empty string? wouldn't the key itself be missing instead?